src/System.Private.ServiceModel/tools/SelfHostedWcfService/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Administration" version="7.0.0.0" targetFramework="net451" userInstalled="true" />
+</packages>
```

Comment:
Question only: Are we using M.W.Administration? If not, can we get rid of it? I don't see references to it in files in this PR, but it might be necessary in the IISHostedWcfService (another reason why I'm not that fond of this cross-directory dependency) :smile: